plugins/org.eclipse.n4js/src/org/eclipse/n4js/n4jsx/ReactHelper.xtend

```diff
@@ -205,7 +220,18 @@ class ReactHelper {
 			return tClassifier;
 		]);
 	}
-	
+
+	def private TClass lookUpReactFragmentComponent(TModule module) {
+		if (module !== null) {
+			for (Type currTopLevelType : module.getTopLevelTypes()) {
+				if (currTopLevelType instanceof TClass
+						&& REACT_FRAGMENT_NAME.equals(currTopLevelType.getName())) {
+					return currTopLevelType as TClass;
+				}
+			}
+		}
+		return null;
```

Comment:
Not sure, but: In case this method returns null, would there be a validation to handle that?